utils/proto/log_replication_streams.proto

```diff
@@ -17,4 +17,5 @@ message VersionString {
 
 message Version {
     string version = 1;
+    bool upgrade = 2;
```

Comment:
let us add a comment explaining what this boolean means.

Also, minor but related to readability - should this flag be called 'is_upgraded'?

Reply:
Modified, thanks!